app/pom.xml

```diff
@@ -129,6 +129,8 @@
     <logback.version>1.2.3</logback.version>
     <micrometer.version>1.2.0</micrometer.version>
     <teiid.version>12.2.1</teiid.version>
+    <aws-java-sdk-core.version>1.11.269</aws-java-sdk-core.version>
+    <aws-dynamodb-local.version>1.11.86</aws-dynamodb-local.version>
```

Comment:
Would it make sense to align to same core version too?

Reply:
Didn't work with same version but now I realized that I included this for the dummy DDB server which I no longer use for testing. So removed!